wallet_ui/components/routes/Authorize.tsx

```diff
@@ -119,7 +119,7 @@ export function Authorize(props: AuthorizeProps) {
   if (agentPrincipal && !agentPrincipal.isAnonymous()) {
     const canisterId = getWalletId();
     const isLocalhost = !!window.location.hostname.match(/^(.*\.)?localhost$/);
-    const canisterCallShCode = `dfx canister --no-wallet${
+    const canisterCallShCode = `dfx canister --wallet=\$(dfx identity get-wallet)${
```

Comment:
```suggestion
    const canisterCallShCode = `dfx canister ${
```
in 0.7.1, the `dfx canister --no-wallet call` and vanilla `dfx canister call` are synonymous. By default, the controller of the cycles wallet will be the user principal, therefore, we shouldn't proxy via `--wallet=<wallet id>`. Instead we can simply remove the redundant flag.